pytorch_lightning/plugins/training_type/ddp_comm_hook_util.py

```diff
@@ -0,0 +1,57 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
I think we should add this under the utilities dir here: https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/utilities/distributed.py

what do you think? @awaelchli ?

Reply:
not Adrian, but I agree, should be moved under the utilities :)

Reply:
Adrian here, yes I agree  that utilities would be a good place considering other ddp related functions are there already :)